scripts/increment_minor.sh

```diff
@@ -45,6 +39,6 @@ if [ -n "$NEEDS_TAG" ]; then
 fi
 
 echo "Updating $VERSION to $NEW_VERSION"
-git tag $NEW_VERSION
+# git tag $NEW_VERSION
```

Comment:
consider adding a dry-run mode (flag / environment) which skips these, since it's a pain to develop and remember to uncomment these before merging

Reply:
fixed and added dry run